airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/YamlSeedConfigPersistenceTest.java

```diff
@@ -49,14 +51,18 @@ public void testGetConfig() throws Exception {
     assertEquals(s3DestinationId, s3Destination.getDestinationDefinitionId().toString());
     assertEquals("S3", s3Destination.getName());
     assertEquals("airbyte/destination-s3", s3Destination.getDockerRepository());
-    assertEquals("https://docs.airbyte.io/integrations/destinations/s3", s3Destination.getDocumentationUrl());
```

Comment:
lol. probably worthwhile to keep this line too. i know it is silly and duplicative but for now the system expects both.

Reply:
Whoops, meant to add not replace. Thanks